egs2/ljspeech/tts2/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set=tr_no_dev_10min
+train_dev=dev_small
+eval_set=eval1
+
+# put your vocoder file and vocoder config file here
+vocoder_file=vocoder/vocoder.pkl
+
+./tts2.sh \
+    --nj 16 \
+    --inference_nj 16 \
+    --fs 16000 --n_shift 320 --n_fft 1280 \
+    --s3prl_upstream_name hubert \
+    --feature_layer 6 \
+    --feature_num_clusters 500 \
+    --lang en \
+    --teacher_dumpdir teacher_dumpdir \
```

Comment:
Maybe this assumed that durations are already extracted somehow (mfa?).
How about adding information about how to run in README or directly adding the processing in this file?